Automattic/pocket-casts-ios
# [Library Update] Automattic-Tracks-iOS

Part of #1366
Fixes #1219

Update `Automattic-Tracks-iOS` (and by doing so, update `Sentry`).

## To test

1. On `AnalyticsEvent` add a `case testTestTest`
1. On `ProfileViewController.swift` `viewDidAppear` add `Analytics.track(.testTestTest)`
2. Run the app and enter/leave Profile a few times
3. Put the app in the background (so events are sent)
4. ✅ Go to [Tracks](https://mc.a8c.com/tracks/live/?eventname=pcios_test_test_test&user=&useragent=) and check that the event was recorded (it might take ~10 minutes\ to appear)

For Sentry we'll need to wait for a build to be released. However, I'm confident it's working because you can see the new format among Sentry releases.

## Checklist
